### DISCUSSION TITLE
Broken/incorrect links on your website

### DISCUSSION THREAD

**Discussion by billytetrud** [requarks/wiki#5420] [Error / Bug Report] [answered]
At the bottom of https://js.wiki/ there are two bad links:

* [Request a New Feature](https://wiki.js.org/feedback)
* [Report a Bug](https://github.com/Requarks/wiki/issues)

The feedback "New Feature" link is just broken. The Report a Bug one links to the github page, which I assume used to have github issues.

**Comment by NGPixel** [comment, marked as answer]
Fixed. Thanks!
